timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/graphql/GraphQlTypesContainer.java

```diff
@@ -52,9 +52,10 @@
   final GraphQLInterfaceType entityInterface;
   final GraphQLInterfaceType valueInterface;
 
-  final TypeResolver objectResolver;
+  final ObjectTypeResolver objectResolver;
+  final ValueTypeResolver valueTypeResolver;
+  final TypeResolver unionTypeResolver;
```

Comment:
Why is unionTypeResolver the only field defined as a TypeResolver and the other two as a specific class?

Reply:
Because I'm using the other two as subtypes and unionTypeResolver only for passing into a method that expects the interface